doc/user_guide/big_data.rst

```diff
@@ -433,6 +433,116 @@ compute the result of all functions that are affected by the axes
 parameters. This is the reason why e.g. the result of
 :py:meth:`~._signals.signal1d.Signal1D.shift1D` is not lazy.
 
+.. _dask_backends:
+
+Dask Backends
+-------------
+
+Dask is a flexible library for parallel computing in Python. All of the lazy operations in
+hyperspy run through dask. Dask can be used to run computations on a single machine or
+scaled to a cluster. The following example shows how to use dask to run computations on a
+variety of different hardware:
+
+Single Threaded Scheduler
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The single threaded scheduler in dask is useful for debugging and testing. It is not
+recommended for general use.
+
+.. code-block:: python
+
+    >>> import dask
+    >>> import hyperspy.api as hs
+    >>> import numpy as np
+    >>> import dask.array as da
+
+    >>> # setting the scheduler to single-threaded globally
+    >>> dask.config.set(scheduler='single-threaded')
+
+Alternatively, you can set the scheduler to single-threaded for a single function call by
+setting the ``scheduler`` keyword argument to ``'single-threaded'``.
+
+Or for something like plotting you can set the scheduler to single-threaded for the
+duration of the plotting call by using the ``with dask.config.set`` context manager.
+
+.. code-block:: python
+
+    >>> s.compute(scheduler="single-threaded")  # uses single-threaded scheduler
+
+    >>> with dask.config.set(scheduler='single-threaded'):
+    >>>     s.plot()  # uses single-threaded scheduler to compute each chunk and then passes one chunk the memory
+
+Single Machine Schedulers
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Dask has three schedulers avilable for single machines.
+
+1. Threaded Scheduler:
+    Fastest to set up but only provides parallelism through threads so only non python functions will be parallelized.
+    This is good if you have largely numpy code and not too many cores.
+2. Processes Scheduler:
+    Each task (and all of the necessary dependencies) are shipped to different processes.  As such it has a larger set
+    up time. This preforms well for python dominated code.
+
+.. code-block:: python
+
+    >>> import dask
+    >>> dask.config.set(scheduler='processes')  # overwrite default with multiprocessing scheduler
+    >>> # Any hyperspy code will now use the multiprocessing scheduler
+    >>> s.compute()  # uses multiprocessing scheduler
+
+    >>> dask.config.set(scheduler='threads')  # overwrite default with threading scheduler
+    >>> # Any hyperspy code will now use the threading scheduler
+    >>> s.compute()  # uses threading scheduler
```

Comment:
@ericpre any idea why this code-block isn't rendering in the documentation?

Reply:
It looks fine to me:
![image](https://github.com/hyperspy/hyperspy/assets/11851990/64681b7f-d3ce-440d-bf14-f4288f893d3d)

Reply:
Weird. It's showing up for me now as well.  Must have been some weird thing with my browser.